### PR TITLE
Added german translation for Addons page

### PR DESCRIPTION
**Pull request by alexzimmer96**
This PR adds a translated version of the Addons page.

### CODE REVIEW

File: content/de/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst) ist eine SDN-Plattform die Policy-Basiertes Networking zwischen Kubernetes Pods und nicht-Kubernetes Umgebungen inklusive Sichtbarkeit und Security-Monitoring bereitstellt.
+* [Romana](http://romana.io) ist eine Layer 3 Network-Lösung für Pod-Netzwerke welche auch die [NetworkPolicy API](/docs/concepts/services-networking/network-policies/) unterstützt. Details zur Installation als kubeadm Add-On sind [hier](https://github.com/romana/romana/tree/master/containerize) verfügbar.
+* [Weave Net](https://www.weave.works/docs/net/latest/kube-addon/) bietet Networking and Network-Policies und arbeitet auf beiden Seiten der Network-Partition ohne auf eine externe Datenbank angwiesen zu sein.
+
+## Service Discovery
+
+* [CoreDNS](https://coredns.io) ist ein flexibler, erweiterbarer DNS-Server der in einem Cluster [installiert](https://github.com/coredns/deployment/tree/master/kubernetes) werden kann und das Cluster-interne DNS für Pods bereitzustellen.
+
+## Visualization &amp; Control
```
Comment (sftim) [inline content/de/docs/concepts/cluster-administration/addons.md]:
```suggestion
## Visualisierung &amp; Kontrol
```
?

File: content/de/docs/concepts/cluster-administration/addons.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+* [NSX-T](https://docs.vmware.com/en/VMware-NSX-T/2.0/nsxt_20_ncp_kubernetes.pdf) Container Plug-in (NCP) bietet eine Integration zwischen VMware NSX-T und einem Orchestator wie z.B. Kubernetes. Außerdem bietet es eine Integration zwischen NSX-T und Containerbasierten CaaS/PaaS-Plattformen wie z.B. Pivotal Container Service (PKS) und OpenShift.
+* [Nuage](https://github.com/nuagenetworks/nuage-kubernetes/blob/v5.1.1-1/docs/kubernetes-1-installation.rst) ist eine SDN-Plattform die Policy-Basiertes Networking zwischen Kubernetes Pods und nicht-Kubernetes Umgebungen inklusive Sichtbarkeit und Security-Monitoring bereitstellt.
+* [Romana](http://romana.io) ist eine Layer 3 Network-Lösung für Pod-Netzwerke welche auch die [NetworkPolicy API](/docs/concepts/services-networking/network-policies/) unterstützt. Details zur Installation als kubeadm Add-On sind [hier](https://github.com/romana/romana/tree/master/containerize) verfügbar.
+* [Weave Net](https://www.weave.works/docs/net/latest/kube-addon/) bietet Networking and Network-Policies und arbeitet auf beiden Seiten der Network-Partition ohne auf eine externe Datenbank angwiesen zu sein.
+
+## Service Discovery
```
Comment (sftim) [inline content/de/docs/concepts/cluster-administration/addons.md]:
```suggestion
## Service-Discovery
```
?

File: content/de/docs/concepts/cluster-administration/addons.md (hunk 1 above)
Comment (alexzimmer96) [reply content/de/docs/concepts/cluster-administration/addons.md]: There is no word "Kontrol" in german. We could use "Kontrolle" or "Überwachung" instead.

Comment (alexzimmer96) [reply content/de/docs/concepts/cluster-administration/addons.md]: There is no word "Kontrol" in german. We could use "Kontrolle" or "Überwachung" instead.

File: content/de/docs/concepts/cluster-administration/addons.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ Die Add-Ons in den einzelnen Kategorien sind alphabetisch sortiert - Die Reihenf
 
 * [CoreDNS](https://coredns.io) ist ein flexibler, erweiterbarer DNS-Server der in einem Cluster [installiert](https://github.com/coredns/deployment/tree/master/kubernetes) werden kann und das Cluster-interne DNS für Pods bereitzustellen.
 
-## Visualization &amp; Control
+## Visualisierung &amp; Überwachung
```
Comment (alexzimmer96) [inline content/de/docs/concepts/cluster-administration/addons.md]: @sftim I think this is better than keeping the heading in english.

Comment (sftim) [reply content/de/docs/concepts/cluster-administration/addons.md]: Fine by me!

**Pull request merged**